UKHomeOffice/nmsw-ui
# NMSW-27 Create confirmation banner for cookie page

# Ticket

NMSW-27

----

## AC
Given I am on the cookie policy page
When I change my cookie preference and click save
Then I am shown a confirmation banner

----

## To test

- run app locally
- go to localhost:3000
- go to cookie page
- change cookie preference selected from 'no' to 'yes'
- click save
- > the page should scroll to the success banner
- > you should see a pane with a success message that you've changed your settings

----

## Notes
